app/routes/articles/ArticleEditRoute.ts

```diff
@@ -19,13 +20,24 @@ const mapStateToProps = (state, props) => {
   const article = selectArticleById(state, {
     articleId,
   });
+  const author = article?.author?.map((e) => {
+    return selectUserById(state, {
+      userId: e,
+    });
+  });
+  console.log(article);
   return {
     article,
     articleId,
     isNew: false,
     initialValues: {
       ...article,
-      ...objectPermissionsToInitialValues(article),
+      ...objectPermissionsToInitialValues({
+        canViewGroups: article.canViewGroups,
+        canEditGroups: article.canEditGroups,
+        canEditUsers: article.canEditUsers,
+        author: author,
```

Comment:
Why is this here? :thinking:

Reply:
Ah, if you need to add `author` here, change the function name to make sense with this change